server/services/database/profiles.js

```diff
@@ -27,7 +27,18 @@ const createProfile = (newProfile) => {
 		.then((result) => result.rows);
 };
 
+const getProfileById = (id) => {
+	return pool
+		.query("SELECT * FROM profiles where id = $1", [id])
+		.then((result) => result.rows[0])
+		.catch((error) => {
+			console.error(error);
+			throw error;
```

Comment:
```suggestion
```
Let's remove this `catch` since we are doing `catch`ing higher up in `server/api/profiles.js`